pkg/awsManager/vpcVpnManager.go

```diff
@@ -274,19 +269,16 @@ func DetachAndDeleteNetworkInterface(client clientpkg.Client, vpcID *string) err
 			break
 		}
 	}
-
 	return nil
 }
 
-func DeleteGateway(client clientpkg.Client, vpcID *string) error {
-
+func deleteGateway(client clientpkg.Client, vpcID *string, logger logr.Logger) error {
```

Comment:
why was this unexported?

Reply:
because it is a helper method that isn't and shouldn't be used outside this file